CONTRIBUTING.adoc

```diff
@@ -145,48 +145,46 @@ TIP: To ensure you get the latest image first run `vagrant box remove fedora_ins
 
         $ vagrant ssh
 
-7.  Run a build:
+7.  You are now ready to edit the source, rebuild and restart OpenShift to test your changes.
```

Comment:
this isn't really a step.  Can you just make it part of the end of "step 6" or maybe "step 5"?